2024-02-18 23:47:22,105 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-18 23:47:22,106 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Configure stats pid to 28164
2024-02-18 23:47:22,106 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-18 23:47:22,106 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-18 23:47:22,106 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-18 23:47:22,108 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-18 23:47:22,108 INFO    MainThread:28164 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240218_234722-w7il6vng\logs\debug.log
2024-02-18 23:47:22,108 INFO    MainThread:28164 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240218_234722-w7il6vng\logs\debug-internal.log
2024-02-18 23:47:22,108 INFO    MainThread:28164 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002DD76B581C0>
2024-02-18 23:47:22,109 INFO    MainThread:28164 [wandb_init.py:init():566] calling init triggers
2024-02-18 23:47:22,109 INFO    MainThread:28164 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'min_samples': 10, 'max_depth': 1}
2024-02-18 23:47:22,109 INFO    MainThread:28164 [wandb_init.py:init():591] re-initializing run, found existing run on stack: ewbqw4z6
2024-02-18 23:47:22,111 INFO    MainThread:28164 [wandb_run.py:_finish():1962] finishing run mc0c0z/DecisionTree/ewbqw4z6
2024-02-18 23:47:22,111 INFO    MainThread:28164 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-02-18 23:47:22,111 INFO    MainThread:28164 [wandb_run.py:_restore():2185] restore
2024-02-18 23:47:22,112 INFO    MainThread:28164 [wandb_run.py:_restore():2191] restore done
2024-02-18 23:47:25,450 INFO    MainThread:28164 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-02-18 23:47:25,450 INFO    MainThread:28164 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-02-18 23:47:25,457 INFO    MainThread:28164 [wandb_run.py:_footer_sync_info():3816] logging synced files
2024-02-18 23:47:25,554 INFO    MainThread:28164 [wandb_init.py:init():616] starting backend
2024-02-18 23:47:25,554 INFO    MainThread:28164 [wandb_init.py:init():620] setting up manager
2024-02-18 23:47:25,556 INFO    MainThread:28164 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-18 23:47:25,557 INFO    MainThread:28164 [wandb_init.py:init():628] backend started and connected
2024-02-18 23:47:25,565 INFO    MainThread:28164 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-02-18 23:47:25,565 INFO    MainThread:28164 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-18 23:47:25,565 INFO    MainThread:28164 [wandb_init.py:init():720] updated telemetry
2024-02-18 23:47:25,567 INFO    MainThread:28164 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-18 23:47:27,119 INFO    MainThread:28164 [wandb_run.py:_on_init():2254] communicating current version
2024-02-18 23:47:27,298 INFO    MainThread:28164 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.16.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-02-18 23:47:27,298 INFO    MainThread:28164 [wandb_init.py:init():804] starting run threads in backend
2024-02-18 23:47:27,383 INFO    MainThread:28164 [wandb_run.py:_console_start():2233] atexit reg
2024-02-18 23:47:27,383 INFO    MainThread:28164 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-02-18 23:47:27,383 INFO    MainThread:28164 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-02-18 23:47:27,384 INFO    MainThread:28164 [wandb_run.py:_redirect():2178] Redirects installed.
2024-02-18 23:47:27,384 INFO    MainThread:28164 [wandb_init.py:init():847] run started, returning control to user process
2024-02-18 23:47:27,384 INFO    MainThread:28164 [wandb_run.py:_config_callback():1342] config_cb None None {'min_samples': 10, 'max_depth': 1}
2024-02-18 23:47:35,295 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-18 23:47:35,295 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Configure stats pid to 28164
2024-02-18 23:47:35,295 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-18 23:47:35,296 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-18 23:47:35,296 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-18 23:47:35,296 INFO    MainThread:28164 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-18 23:47:35,296 INFO    MainThread:28164 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240218_234735-e0g3eu4x\logs\debug.log
2024-02-18 23:47:35,297 INFO    MainThread:28164 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240218_234735-e0g3eu4x\logs\debug-internal.log
2024-02-18 23:47:35,297 INFO    MainThread:28164 [wandb_init.py:init():566] calling init triggers
2024-02-18 23:47:35,297 INFO    MainThread:28164 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'min_samples': 10, 'max_depth': 6}
2024-02-18 23:47:35,297 INFO    MainThread:28164 [wandb_init.py:init():591] re-initializing run, found existing run on stack: w7il6vng
2024-02-18 23:47:35,299 INFO    MainThread:28164 [wandb_run.py:_finish():1962] finishing run mc0c0z/DecisionTree/w7il6vng
2024-02-18 23:47:35,299 INFO    MainThread:28164 [jupyter.py:save_history():445] not saving jupyter history
2024-02-18 23:47:35,299 INFO    MainThread:28164 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-18 23:47:35,299 INFO    MainThread:28164 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-18 23:47:35,299 INFO    MainThread:28164 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-02-18 23:47:35,299 INFO    MainThread:28164 [wandb_run.py:_restore():2185] restore
2024-02-18 23:47:35,300 INFO    MainThread:28164 [wandb_run.py:_restore():2191] restore done
2024-02-18 23:47:38,841 INFO    MainThread:28164 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-02-18 23:47:38,841 INFO    MainThread:28164 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-02-18 23:47:38,846 INFO    MainThread:28164 [wandb_run.py:_footer_sync_info():3816] logging synced files
